.travis.yml

```diff
@@ -147,8 +147,9 @@ after_success:
       lcov --version
       pwd && ls -laF . /usr/bin/gcov*
 
-      cd $BUILD_ROOT && pwd
-      find ~/ -name "*gcno" -or -name "*gcda"
+      # cd $BUILD_ROOT && pwd
+      cd $SOURCE_ROOT && pwd
```

Comment:
I'm unclear about this change. Looks like travis is building now with `cd $BUILD_ROOT` but you're saying there's a problem. I'm not seeing a problem though. Where can I find it so I can clear up my confusion about this? Thanks!!!